helm_deploy/hmpps-interventions-service/templates/_envs.tpl

```diff
@@ -34,6 +34,12 @@ env:
   - name: COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_TEAMCODE
     value: "{{ .Values.env.COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_TEAMCODE }}"
 
+  - name: APPLICATION_INSIGHTS_IKEY
+    valueFrom:
+      secretKeyRef:
+        name: {{ template "app.name" . }}
```

Comment:
This would look up the secret from `hmpps-interventions-service` secret, is that correct?